fmn/consumer/__init__.py

```diff
@@ -1,3 +1,23 @@
+"""
+The FMN consumer package contains all the service that process the incoming
+`fedmsg`_ messages. The messages are processed in three steps:
+
+1. The `fedmsg consumer`_ defined in :mod:`fmn.consumer.consumer` subscribes
+   to every fedmsg topic (``*``). It places these messages into a RabbitMQ
+   message queue called ``worker``. This message broker provides message
+   durability for FMN as it processes the messages.
+
+2. One or more :mod:`fmn.consumer.worker` processes are started and consume
+   messages from the ``worker`` message queue. These worker processes take
+   each message and determine who should receive notifications based on their
+   message filters. It then records this information in the message and
+   publishes the message to the ``backend`` queue.
+
+3. A single :mod:`fmn.consumer.backend` process is run and is responsible for
```

Comment:
Isn't twisted supposed to allow running more than one?

I know i've wanted to use only one for simplicity reasons but it was my understanding that we could scale (which is also a reason why the service is written as a template not a single service)